AdvanceDS/Assignment 1/genralize list/GenralizeList.java

```diff
@@ -0,0 +1,228 @@
+package addsAssignment;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class GenralizeList {
+	private Node start;
+	private Stack expression;
+	private int max = 0;
+	private int sum = 0;
+	private String outputList = "";
```

Comment:
At least the getter was expected to get the generated list. May that's why a test for the constructor is missing.